js/utils/templateHelpers.js

```diff
@@ -1,5 +1,10 @@
-// todo: add in some helpers!
+import app from '../app';
+
+const polyT = function (...args) {
+  return app.polyglot.t(...args);
+};
+
+export {
```

Comment:
Can we directly export the polyT function?

```
export function plyT(...args) {
  return app.polyglot.t(...args);
}
```

In ES6 modules, you can export multiple things individually, for example:

```
export function a() {};

export function b() {};

const variableA = 'meatballs';

export { variableA };
```

The benefit is that then individual items can be imported and only those would go into memory. If I'm not mistaken, if you just export everything in one big object, I think the whole thing gets brought in.